tests/acceptance/request_test.py

```diff
@@ -99,6 +114,7 @@ def test_400_if_required_body_is_missing(test_app):
     ).status_code == 400
 
 
+@pytest.mark.xfail
```

Comment:
Ok, I'll add. Do we want to be very strict, and also require the content-type value for json as well?